### ISSUE TITLE
Fire up the pulp mill!

### ISSUE THREAD

**Issue by joeld42** [NaNoGenMo/2016#45]
I've always wanted to do something like this. My theory is that a bunch of dumb ideas stuck together will create great things. Or at least something entertaining. My project is "pulpmill", which I hope will create trashy epic fantasy novels. Starting with of course generating the map and world-building.  

Best place to follow for updates is on twitter at @joeld42, my git repo for this is
https://github.com/joeld42/jbd_nanogenmo

**Comment by hugovk** [comment]
Looking good!
https://twitter.com/joeld42/status/792221481975554048
https://twitter.com/joeld42/status/792504273602871296
https://twitter.com/joeld42/status/792641233726300160
https://twitter.com/joeld42/status/792643411991932928
https://twitter.com/joeld42/status/792991028433170432

Please do update from here too from time to time, and use #nanogenmo on Twitter too :)

**Comment by joeld42** [comment]
I'll try! And I'll start using the #nanogenmo tag.

Added pruning for the roads, have placeholder dungeons (but without interesting names yet)
https://twitter.com/joeld42/status/793493312350986240

**Comment by joeld42** [comment]
Generating story paths:
https://twitter.com/joeld42/status/793721777784356864

Dungeon Names:
https://twitter.com/joeld42/status/793855250432012289

**Comment by joeld42** [comment]
Update: It's mostly working, right now it generates about 2000 word novels but I can just make an 
"anthology" :)

Here's a pretty representative preview:
https://twitter.com/joeld42/status/803320977316683776

Going to spend another day or two cleaning up some placeholder garbage, but then again I don't think I ever turn anything in on time so...

Here's some more covers since they are so much fun:
https://twitter.com/joeld42/status/803091627795353600
https://twitter.com/joeld42/status/803079660254863360
https://twitter.com/joeld42/status/803044877638373376

I'll write up a blog post with some details sometime in December.

**Comment by hugovk** [comment]
They look great! I think late submissions are fine :)

**Comment by joeld42** [comment]
And it's done! Well, there's a lot more I'd like to improve of course, but it's done enough. The novels tend to turn out around 2.5-3k words each so to meet the 50k word requirement I generated an "anthology" with 17 novels for a total of 52,087 words. That's more fun anyways because you get more covers and more heroes.

Result can be found here:
https://github.com/joeld42/jbd_nanogenmo/raw/master/nanogenmo2016/sample/jbd_nanogenmo2016.pdf

Thanks everyone!

**Comment by hugovk** [comment]
Labelled, and congrats!

An anthology is good because then we get more of the amazing covers!

**Comment by aparrish** [comment]
I just want to say (years too late) that this is *fantastic*. "It's the Bone Cube."

**Comment by greg-kennedy** [comment]
Yeah I stumbled across this thanks to the write-up in your blog, and it's awesome.  The presentation is excellent, the covers and map generation... there's a TON of templates making it fun to read... just, really great work.
